### ISSUE TITLE
"wait: no child process" fails build + takes app offline

### ISSUE THREAD

**Issue by contra** [convox/rack#1171]
Did a usual deploy, got this:

<img width="473" alt="screen shot 2016-08-31 at 11 29 53 am" src="https://cloud.githubusercontent.com/assets/425716/18138204/999812c2-6f79-11e6-9d2d-489c9fe5c2f3.png">

After this error happened, ELB started returning 503s and the application was completely offline + didn't restart or recover without manual intervention.

Rack Versions tested: `20160827184744`, `20160901022059`

docker-compose.yml:
```yaml
web:
  build: .
  command: node /usr/src/app/bin/server
  labels:
    - convox.port.443.protocol=https
    - convox.port.80.protocol=http
    - convox.health.port=3000
    - convox.health.path=/v1/tables
    - convox.deployment.minimum=25
    - convox.deployment.maximum=150
  environment:
    - PORT=3000
    - NODE_ENV
  ports:
    - 80:3000
    - 443:3000
  volumes:
    - .:/usr/src/app
```

**Comment by contra** [comment]
Hmm, I think the issue is:

```yaml
labels:
    - convox.deployment.minimum=25
    - convox.deployment.maximum=150
```

Removing it clears up this issue.

**Comment by mattmanning** [comment]
Good hypothesis, but we are also seeing another user hit this error with a `docker-compose.yml` with no deployment labels.

```
$ cat docker-compose.yml 
main: 
build: . 
  ports: 
    - 2878 
    - 2003
```

**Comment by contra** [comment]
@mattmanning Yeah, I just started getting it again without those labels. I guess i got lucky once out of the latest hundred or so attempts to deploy code.

**Comment by ddollar** [comment]
This should have been fixed by #1184
